exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/DirectRowSet.java

```diff
@@ -141,5 +136,22 @@ public SingleRowSet toIndirect(Set<Integer> skipIndices) {
   }
 
   @Override
-  public SelectionVector2 getSv2() { return null; }
+  public SelectionVector2 getSv2() {
+    return null;
+  }
+
+  public static class RowSetWriterBuilder extends BaseWriterBuilder {
```

Comment:
See note above about cluttering the project with many small classes used in only one location.